src/wrap_mempool.cpp

```diff
@@ -393,8 +617,65 @@ void pyopencl_expose_mempool(py::module &m)
     typedef pooled_buffer cls;
     py::class_<cls, /* boost::noncopyable, */
       pyopencl::memory_object_holder>(
-          m, "PooledBuffer"/* , py::no_init */)
+          m, "_tools_PooledBuffer"/* , py::no_init */)
+      .def("release", &cls::free)
+      // undocumented for now, for consistency with SVM
+      .def("bind_to_queue", [](cls &self, pyopencl::command_queue &queue) { /* no-op */ })
+      .def("unbind_from_queue", [](cls &self) { /* no-op */ })
+      ;
+  }
+
+#if PYOPENCL_CL_VERSION >= 0x2000
+  {
+    typedef svm_allocator cls;
+    py::class_<cls, std::shared_ptr<cls>> wrapper(
+        m, "_tools_SVMAllocator");
+    wrapper
+      .def(py::init<std::shared_ptr<pyopencl::context>  const &, cl_uint, cl_uint, pyopencl::command_queue *>(),
+          py::arg("context"),
+          py::arg("alignment"),
```

Comment:
I suggest providing a default value of `0` for alignment.

Reply:
Done in https://github.com/inducer/pyopencl/compare/00a72d1c2f2f50444dac8824b30c82418660fe76..d90fff1ff2fdd0b071a5bf85cf88457bb283336f.